packages/forma-36-react-components/src/components/Portal/Portal.tsx

```diff
@@ -6,7 +6,7 @@ export interface PortalProps {
   container?: Document | HTMLElement;
 }
 
-export default function Portal({
+export function Portal({
```

Comment:
Looking good! 👏 Maybe we would be able to write README with examples of usage? 🙂

Reply:
Ah yes, of course, thanks for reminding me 🙂 I will add this.